Tests/TuistKitTests/Services/Project/ProjectShowServiceTests.swift

```diff
@@ -0,0 +1,152 @@
+import Foundation
+import MockableTest
+import TuistCore
+import TuistLoader
+import TuistServer
+import TuistSupport
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistKit
+
+final class ProjectShowServiceTests: TuistUnitTestCase {
+    private var opener: MockOpening!
+    private var configLoader: MockConfigLoading!
+    private var serverURLService: MockServerURLServicing!
+    private var getProjectService: MockGetProjectServicing!
+    private var subject: ProjectShowService!
+
+    override func setUp() {
+        super.setUp()
+        opener = MockOpening()
+        configLoader = MockConfigLoading()
+        serverURLService = MockServerURLServicing()
+        getProjectService = MockGetProjectServicing()
+        subject = ProjectShowService(
+            opener: opener,
+            configLoader: configLoader,
+            serverURLService: serverURLService,
+            getProjectService: getProjectService
+        )
+
+        given(serverURLService)
+            .url(configServerURL: .any)
+            .willReturn(Constants.URLs.production)
+    }
+
+    override func tearDown() {
+        opener = nil
+        configLoader = nil
+        subject = nil
```

Comment:
There are some variables that are not `nil`'ed on `tearDown`
```suggestion
        subject = nil
        serverURLService = nil
        getProjectService = nil
        subject = nil
```